### PR TITLE
Add flying skulls

### PR DESCRIPTION
**Pull request by SariusSkelrets**
#### Summary

Content: Add flying skulls

#### Purpose of change

The winged zombies from LyleSY in [#42768](https://github.com/CleverRaven/Cataclysm-DDA/pull/42768) made me want some kind of airborne zombie. After I-am-Erk ajusted them in [#43188](https://github.com/CleverRaven/Cataclysm-DDA/pull/43188), I searched how to add flying zombies to the game and I found how.

By removing the excessive weight of some zombies by decapitation and by developing wings with the skull zombie's ears...

(a wierd case of zombie evolution that I leave for a later PR where the skull zombie or the zombie between it and the flying skull become a headless zombie while creating the flying skull. For now I just make them appearing out of nowhere before doing something as complicated)

...I created a lightweight zombie who is basically an head with wings

 Its low weight allow it to fly instead of gliding but there's many downsides for that zombie.

1: It have low hp and no armor
2: It can't bash (if it can bash someday that will stun/damage it)
3: It can only bite (this one is more of an upside as biting can infect the player)
4: not aerodynamical at all as the head cause drag (slow speed but still able to fly)

#### Describe the solution

Adding the flying skull

#### Describe alternatives you've considered

Not adding the flying skull
#### Testing

Spawning it ingame and fixing the crash that occured because i forgot to add the material tag

#### Additional context

I plan much more variants of the flying skull as a big-jawed and a shady variant or something halfways between the flying skull and the zombie brute: the crawling face.

### CODE REVIEW

**Comment by Maleclypse** [comment]
to fix PR test  SUMMARY: Content "Add Flying Skulls"

**Comment by I-am-Erk** [comment]
I... why, though?

**Comment by Rivet-the-Zombie** [comment]
Okay, so hear me out: *flaming* flying skulls...

...Or maybe I just play too much Doom.

File: data/json/monsters/zed_misc.json [hunk 1]
```diff
@@ -1063,6 +1063,49 @@
       "FILTHY"
     ]
   },
+  {
+    "id": "mon_flying_skull",
+    "type": "MONSTER",
+    "name": { "str": "flying skull" },
+    "//": " The idea is making some zombies growing wings and the needed articulations/muscles/ect. on the sides of their heads before severing their necks to free the flying skull then grow another head to repeat the process.",
+    "description": "An human skull with two leathery wings on the sides of its head.  Devoid of skin, it flies aimlessly, searching something to bite.",
```
Comment (matskuman5) [inline data/json/monsters/zed_misc.json:1071]:
```suggestion
    "description": "A human skull with two leathery wings on the sides of its head.  Devoid of skin, it flies aimlessly, searching for something to bite.",
```

**Comment by SariusSkelrets** [comment]
@Rivet-the-Zombie just add them the flammable flag them throw a molotov and voila! Flaming flying skulls

**Comment by jeremyshannon** [comment]
So basically like a Vargoiulle? That's pretty metal.

**Comment by SariusSkelrets** [comment]
I didn’t knew about vargouilles before youmake the comparison but yeah, skinless vargouilles are almost identical to how i see the flying skull

**Comment by I-am-Erk** [comment]
So like, I was being a little facetious, but I am not clear why or how this would exist either from a lore or a gameplay perspective.

**Comment by SariusSkelrets** [comment]
From a lore perspective the flying skull could be an attempt made by severed heads to move again (some grow legs, other get wings) or a result of the infinite mutative potential of the blob as the skull zombie’s ears morph into ears and it lose its head someway or the body atrophy until the head breaks free or until the body becomes light enough to be lifted by the wings

Why the human zombies stays with a humanoid form? Why there’s no zombies with more or less limbs? Deformed ones as flesh and bones grow on their bodies?  Why zombies always become more dangerous as time pass without one or two of them who weren’t lucky like that hulk unable to move under the weight of its own muscles?
The flying skulls are one of these wierd post-cataclysm things that people see and ask themselves some questions before shooting said flying skull

Not all mutations are good so some unable-to-fly flying skulls (either by deformed or misplaced wings) should exist too as the blob can’t just create functional special zombies everytime. 

Other types of zombies crippled by mutations would add to the «uncontrollable and unpredictable zombies mutations» aspect of the blob but that’s will be left for another PR

From a gameplay perspective, the flying skulls are , hit-and-run, flying and able to infect the player but their main threat is their ability to reach the player on roofs or when they drive an helicopter. 

IIRC there’s a minimal weight required for the blob to reanimate an organism so zombie birds are more than unlikely and since zombies can’t climb or antpile their way to the roof how they can reach the PC without flying or destroying the entire building?

So the flying skulls serve to attack the PCs hiding in a flying machine, on a roof or behind hulkproof walls while expanding the zombie faction

**Comment by aoae23** [comment]
This is the kind of body horror zombie evolution we need.

**Comment by Aphegis** [comment]
Just my opinion: I never liked this idea of skulls that fly, your proposed skull at least have some type of wings, but at lest for me it would be too fantasy. If the majority agrees with me, maybe this should be included in aftershock??

**Comment by SariusSkelrets** [comment]
As i said it’s supposed to be the result of the blob mutations taking a turn nobody understand instead of another «become the thing you were before but better». The blob is supposed to be able to mutate without limits but the human zombies never stops from being humanoid without becoming a dissoluted devourer or a Jabberwock. 

If it can’t go into the main game, I’ll be glad to see it in/as a mod but i’ll wait because nothing is certain for now about the flying skull

**Comment by John-Candlebury** [comment]
Not in aftershock, there's nothing cyberpunk sci-fi about this. And I wouldnt want Warhammer 40k style servo skulls either.

**Comment by jeremyshannon** [comment]
If the severed head has enough mass to reanimate, I don't see why the blob wouldn't at least try to make something out of it. And how that could itself be kind of horrific. (Thinking about it I'm reminded of a moment in John Carpenter's The Thing.)

**Comment by LyleSY** [comment]
The image is great. As a standalone idea I'm not sure it fits, but I see where you're going with it. An intermediate stage would help, a zombie evolution, especially for something with no evolutions at all right now like the scarred zombie, especially if you incorporated some more bee characteristics like a stinger with VENOM. Look at what Erk is doing with his flesh-raptors as a good example.
If it doesn't fit in Main, I could see it working well with one of the magical themed mods.

**Comment by SariusSkelrets** [comment]
As i said its an «evolution» of the skull zombie who’s described as something with enormous ears (it’s an «evolution» because the flying skull is created and the skull zombie become an headless one) but it could also be the result of already crippled zombies. 
You surely remember that «cut the head or destroy the brain» thing and people during the cataclysm remembered it too causing large amounts of severed heads and headless corpses. 
With all the heads everywhere some have grown legs and other wings to move again.
A zombie evolution between the zombie and my flying skull could be one getting its gigantic ears progressively transformed in batwings

**Comment by SariusSkelrets** [comment]
After reading (#43188)[https://github.com/CleverRaven/Cataclysm-DDA/pull/43188] another time I noticed that the raptors were airborne instead of another walking zombie as the name made me though so the skull zombies’ severed flying heads aren’t needed because the raptors do the same thing

I’ll plan of adding some cases where zombie evolution wasn’t as good as supposed like that too-muscled-to-move hulk to diversify what the blob can do because as of now zombie mutations are never harmful of crippling for them

**Comment by SariusSkelrets** [comment]
I’m closing the pull request because I guess the winged heads won’t be added in the game. 
I must admit, even if my creation isn’t accepted, all the process from creating it to making the pull request and anything between was very fun for me so I’ll try to find something else I could do a PR with.

**Pull request closed without merging**